Update to LocationComponent styling option charts writing (#976)
mapbox/android-docs: src/pages/maps/overview/location-component.md

| XML Option | Explanation |
| --- | --- |

> | mapbox_foregroundDrawable | Drawable image which would replace the blue circle. |

Reviewer: ```suggestion
| mapbox_foregroundDrawable | Drawable image that would replace the blue circle. |
```

| mapbox_foregroundTintColor | The blue circle's color. |

> | mapbox_backgroundDrawable | Drawable image which would replace the white circle.  |

Reviewer: ```suggestion
| mapbox_backgroundDrawable | Drawable image that would replace the white circle.  |
```

| mapbox_backgroundTintColor | The white circle. |

> | mapbox_bearingDrawable | Drawable image which would replace the blue triangle. |

Reviewer: ```suggestion
| mapbox_bearingDrawable | Drawable image that would replace the blue triangle. |
```

| XML Option | Explanation |
| --- | --- |

> | mapbox_foregroundDrawableStale | Drawable image which would replace where the grey circle is. |

Reviewer: ```suggestion
| mapbox_foregroundDrawableStale | Drawable image that would replace the grey circle's location. |
```

| mapbox_foregroundStaleTintColor | The grey circle's color. |

> | mapbox_backgroundDrawableStale | Drawable image which would replace where the white circle is. |

Reviewer: ```suggestion
| mapbox_backgroundDrawableStale | Drawable image that would replace the white circle's location. |
```

> | mapbox_backgroundDrawableStale | Drawable image which would replace where the white circle is. |

Reviewer: ```suggestion
| mapbox_backgroundDrawableStale | Drawable image that would replace the white circle's location. |
```
